linux_os/guide/system/auditing/auditd_configure_rules/audit_unsuccessful_file_modification/audit_rules_unsuccessful_file_modification_open_rule_order/oval/shared.xml

```diff
@@ -1,12 +1,13 @@
 <def-group>
   <definition class="compliance" id="audit_rules_unsuccessful_file_modification_open_rule_order" version="1">
     <metadata>
-      <title>Ensure auditd Collects Detailed Information on Unsuccesful Access Attemps to Files</title>
+      <title>Ensure auditd Unauthorized Access Attempts To open Are Ordered Correctly</title>
```

Comment:
This is strange wording. What about this?
`Ensure auditd Rules For Unauthorized Attempts To open Are Ordered Correctly`